drivers/i2c/Kconfig

```diff
@@ -130,6 +130,11 @@ config I2C_STM32_INTERRUPT
 	help
 	  Enable Interrupt support for the I2C Driver
 
+config I2C_STM32_COMBINED_INTERRUPT
```

Comment:
Do we really need this option? If I2C combined interrupt is supported only on F0 SoCs, could we use the F0 family flag in its place?

Reply:
At least, STM32L0 also uses a combined interrupt line, and I did the same with SPI FIFO to simplify the driver code and put the logic in Kconfig instead

Reply:
Ok, then we should keep it